fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/ModNioResourcePack.java

```diff
@@ -51,9 +53,9 @@
 import net.fabricmc.loader.api.ModContainer;
 import net.fabricmc.loader.api.metadata.ModMetadata;
 
-public class ModNioResourcePack extends AbstractFileResourcePack implements ModResourcePack {
+public class ModNioResourcePack implements ResourcePack, ModResourcePack {
 	private static final Logger LOGGER = LoggerFactory.getLogger(ModNioResourcePack.class);
-	private static final Pattern RESOURCE_PACK_PATH = Pattern.compile("[a-z0-9-_.]+");
+	private static final Pattern RESOURCE_PACK_PATH = Pattern.compile("[a-z\\d-_.]+");
```

Comment:
Why this change? Won't this allow non-ASCII paths? (like Persian numerals)

Reply:
Idea said it was equivalent

Reply:
...why would you change it then? 🤔

Reply:
You mean why I reverted that change?
It's not really related to this PR and idea might just be recommending the change for international compatibility

Reply:
I was wondering more about why you changed it originally since the ascii pattern is perfectly clear, but yeah it makes sense to revert it here